src/main/java/org/stellar/sdk/responses/LiquidityPoolResponse.java

```diff
@@ -0,0 +1,149 @@
+package org.stellar.sdk.responses;
+
+import com.google.common.base.Objects;
+import com.google.gson.annotations.SerializedName;
+
+import org.stellar.sdk.Asset;
+import org.stellar.sdk.LiquidityPoolID;
+import org.stellar.sdk.xdr.LiquidityPoolType;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.errorprone.annotations.ForOverride;
+
+/**
+ * Represents liquidity pool response.
+ * @see <a href="https://www.stellar.org/developers/horizon/reference/resources/liquiditypool.html" target="_blank">Liquidity Pool documentation</a>
+ * @see org.stellar.sdk.requests.LiquidityPoolsRequestBuilder
+ * @see org.stellar.sdk.Server#liquidityPools()
+ */
+public class LiquidityPoolResponse extends Response {
+  @SerializedName("id")
+  private LiquidityPoolID id;
+  @SerializedName("paging_token")
+  private String pagingToken;
+  @SerializedName("fee_bp")
+  private Integer feeBP;
+  @SerializedName("type")
+  private LiquidityPoolType type;
+  @SerializedName("total_trustlines")
+  private String totalTrustlines;
+  @SerializedName("total_shares")
+  private String totalShares;
+  @SerializedName("reserves")
+  private Reserve[] reserves;
+  @SerializedName("_links")
+  private Links links;
+
+  LiquidityPoolResponse(LiquidityPoolID id) {
+    this.id = id;
+  }
+
+  public LiquidityPoolID getID() {
+    return id;
+  }
+
+  public String getPagingToken() {
+    return pagingToken;
+  }
+
+  public Integer getFeeBP() {
+    return feeBP;
+  }
+
+  public LiquidityPoolType getType() {
+    return type;
+  }
+
+  public String getTotalTrustlines() {
+    return totalTrustlines;
+  }
+
+  public String getTotalShares() {
+    return totalShares;
+  }
+
+  public Reserve[] getReserves() {
+    return reserves;
+  }
+
+  /**
+   * Represents liquidity pool reserves.
+   */
+  public static class Reserve {
+    @SerializedName("amount")
+    private final String amount;
```

Comment:
same comment about Long instead of String

Reply:
But this is a float...

Reply:
yeah, sorry about that. I forgot that they're actual floating point values